docs/integration/code-adaptions.mdx

```diff
@@ -49,13 +51,22 @@ private void AutoConnect_OnServerReady()
 <TabItem value="js" label="JavaScript">
 
 ```js
-alert("TODO");
+// JS has no native Random seeding, you must use a package
+// on the client and server like https://github.com/davidbau/seedrandom
+
+const Server = new ArcadeServerSDK();
+
+// ... initialization
+
+const status = Server.getServerStatus();
```

Comment:
This needs to be run in a retry loop or the SDK needs to handle that. For unity that's handled in the AutoConnect behavior.

Reply:
Also:
```suggestion
const status = await Server.getServerStatus();
```

Reply:
I'll have the SDK handle it

Reply:
I thought you said we did not have to retry anything that talks over localhost though? Or is this because the server might not be ready?

Reply:
nvm I see